### PR TITLE
Man page should correspond to current commands

### PR DESCRIPTION
**Pull request by jakubjedelsky**
Edit some details and add info for play and upload commands.

### CODE REVIEW

**Comment by sickill** [comment]
I completely forgot about the man page :/
I have no knowledge of (g)roff unfortunately so this was always something on my to-do list that wast buried by dust. I wonder if there's some tool that could automatically generate man page out of for example markdown.

**Comment by sickill** [comment]
Just found this: https://rtomayko.github.io/ronn/ronn.1.html
Not sure how useful it would be though.

File: man/asciinema.1 [hunk 1]
```diff
@@ -39,20 +35,33 @@ Recording finishes when you exit the shell (hit \fBCtrl+D\fP or type \fIexit\fP)
 \-c
```
Comment (sickill) [inline man/asciinema.1]: One thing here: `-c` has now `--comand` alternative.

**Comment by jakubjedelsky** [comment]
It will be cool to generate it automatically! But there are few things which, I think, shouldn't be in man page (install and build info). But the true is it would be much more simple. I don't know ronn tool, but can take care about man page..

**Comment by sickill** [comment]
I was thinking about cutting out some part(s) of readme with something like awk, piping it to ronn. We can definitely look into that some day.

**Comment by sickill** [comment]
Thanks! :heart:

**Pull request merged**